massgov/mayflower
# DP-15262: Update Info Details sidebar handling

## Description
Updated JS and twig template handling for sidebar content on the Info Details page.
**The sidebar JS update will affect other pages, but is a basic existence check and should be low risk.**

## Related Issue / Ticket
- [JIRA issue](https://jira.mass.gov/browse/DP-15262)

## Steps to Test

1. Checkout `patternlab/DP-15262--info-detail-sidebar` into your local mayflower instance.
2. If necessary, delete the `mayflower-dev` folder from your Drupal instance under `docroot/libraries/mayflower-dev`
3. In your Mayflower instance, run `gulp` to build the dev copy of Mayflower for Drupal to use.
4. Visit `/info-details/ebt-card-to-culture-organizations`
5. Note that the first section under the main content, labeled "Central Region" is displayed full-width as requested.

## Screenshots
<img width="1101" alt="DP-15262--info-details-no-sidebar" src="https://user-images.githubusercontent.com/6147609/65198640-16195880-da41-11e9-8461-1629c7737b40.png">

#### Impacted Areas in Application
- Info Details template.
- Any page using the imageFill.js file to dynamically render elements that are below the sidebar as full-width.
